### PR TITLE
Slash and burn pass on loadout.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
- Rewrites loadout to use decls.
- Removes all job and faction loadout choices pending attaching them specifically to their jobs.
- Removes storage items, modular computers, the hand labeller, and several loot items from loadout.
- Adds loadout support for per-category and culture selection based restrictions.

This is a first-pass towards a loadout system oriented more on the original idea, ie. purely aesthetic choices for a character. Job and faction gear will be associated with jobs and factions rather than loadout.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
DNM while I work out why loading isn't working.

File: code/modules/client/preference_setup/loadout/loadout.dm [hunk 1]
```diff
@@ -326,14 +309,21 @@ var/global/list/gear_datums = list()
...
 
-/datum/gear/New()
+/decl/loadout_option/Initialize()
+	. = ..()
+
+	if(display_name && (!global.using_map.loadout_blacklist || !(type in global.using_map.loadout_blacklist)))
+		global.gear_datums[display_name] = src
+		var/decl/loadout_category/LC = GET_DECL(category)
+		LC.gear[display_name] = src
+		LC.gear = sortTim(LC.gear, /proc/cmp_text_asc)
```
Comment (afterthought2) [inline code/modules/client/preference_setup/loadout/loadout.dm]: Isn't there a sorted insert helper instead?

Comment (MistakeNot4892) [reply code/modules/client/preference_setup/loadout/loadout.dm]: No idea.

**Comment by MistakeNot4892** [comment]
Beh I broke this while trying to centralize the cost calc. Will fix it tomorrow.

**Comment by MistakeNot4892** [comment]
Seems to be working properly now.

**Comment by MistakeNot4892** [comment]
Ready for review.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**